organisations/tests/test_api.py

```diff
@@ -909,3 +915,147 @@ def test_person_study_groups_unauthorized(
         variables={"id": to_global_id("PersonNode", person.id)},
     )
     assert executed["data"]["person"]["studygroupSet"]["edges"] == []
+
+
+@pytest.mark.django_db
+class PublicPersonNodeTests:
+    @pytest.fixture
+    def client(self):
+        return Client(schema=test_schema)
+
+    @pytest.fixture
+    def users(self):
+        active_user = UserFactory(username="active", is_staff=True)
+        inactive_user = UserFactory(username="inactive", is_staff=False)
+        return {"active": active_user, "inactive": inactive_user}
+
+    @pytest.fixture
+    def persons(self, users):
+        public_person1 = Person.objects.create(
+            name="Public Person One",
+            phone_number="111-111-1111",
+            email_address="public1@example.com",
+            user=None,
+        )
+        public_person2 = Person.objects.create(
+            name="Public Person Two",
+            phone_number="222-222-2222",
+            email_address="public2@example.com",
+            user=users["active"],
+        )
+        private_person = Person.objects.create(
+            name="Private Person",
+            phone_number="333-333-3333",
+            email_address="private@example.com",
+            user=users["inactive"],
+        )
+        return {
+            "public1": public_person1,
+            "public2": public_person2,
+            "private": private_person,
+        }
```

Comment:
How about adding a person without linked user i.e. one of the filtered out cases in PublicPersonNode.get_queryset?
```suggestion
        private_person1 = Person.objects.create(
            name="Private Person One",
            phone_number="333-333-3333",
            email_address="private1@example.com",
            user=users["inactive"],
        )
        private_person2 = Person.objects.create(
            name="Private Person Two",
            phone_number="444-444-4444",
            email_address="private2@example.com",
            user=None,  # No linked user
        )
        return {
            "public1": public_person1,
            "public2": public_person2,
            "private1": private_person1,
            "private2": private_person2,
        }
```

And
```
        assert "Private Person" not in names
```
to
```
        assert "Private Person One" not in names
        assert "Private Person Two" not in names
```
later?